Sources/RxCBCentral/RxPeripheral.swift

```diff
@@ -80,6 +80,10 @@ public protocol RxPeripheral: AnyObject {
     /// The data returned will be processed by the `Preprocessor` given when registering
     /// for notifications for this `characteristic`, if one was provided.
     func notificationData(for characteristic: CBUUID) -> Observable<Data>
+    
+    /// Validate if service exist
+    func hasService(service: CBUUID) -> Observable<Bool>
```

Comment:
Nice addition, can you use modify this interface to use more Swift style naming, like:

`func hasService(_ service: CBUUID) -> Observable<Bool>`